projects/Rockchip/devices/OdroidGoAdvance/packages/odroidgoa-utils/system.d/odroidgoa-hotkeys.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Odroid Go Advance Hotkeys
+ConditionPathExists=/dev/input/event0
+ConditionPathExists=/dev/input/event2
+Before=retroarch.target
+
+[Service]
+Environment=HOME=/storage
+EnvironmentFile=/storage/.config/emulationstation/scripts/es_env.sh
```

Comment:
What is this?

Reply:
Looks like emulationstation executable script

Reply:
i have missed this